terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -167,6 +179,22 @@ resource "kubernetes_cron_job" "workflow_manager" {
               args = [
                 "--k8s-namespace", var.kubernetes_namespace,
                 "--input-bucket", var.ingestion_bucket,
+                "--bsk-secret-name", kubernetes_secret.batch_signing_key.metadata[0].name,
+                "--pdks-secret-name", var.packet_decryption_key_kubernetes_secret,
+                "passthrough",
+                # These args are passed to facilitator in addition to the per-batch
+                # args that workflow-manager builds to pass in.
+                "intake-batch",
+                "--aggregation-id", "fake-1",
+                "--instance-name", "foo",
+
+                "--ingestor-input", var.ingestion_bucket,
+                "--ingestor-identity", var.ingestion_bucket_role,
+                "--ingestor-manifest-base-url", "I dunno",
```

Comment:
This has to be plumbed in from main.tf. I am just tinkering with a branch where I wire all this into a config map.